plexupdate.sh

```diff
@@ -548,6 +548,11 @@ if [ ! -z "${PLEXSERVER}" -a "${AUTOINSTALL}" = "yes" ]; then
 fi
 
 if [ "${AUTOINSTALL}" = "yes" ]; then
+	if ! hash ldconfig 2>/dev/null && [ -x /sbin/ldconfig -a "${DISTRO}" = "ubuntu" ]; then
```

Comment:
Oh, that's a leftover from when I first started writing the patch. The original version tried much harder to do the right thing in every case but it turned into a 30+ line block of code pretty quickly. When I simplified it to just allow dpkg to error out anyway, I forgot to take it out. I can pull it out if you want to keep the code slightly cleaner.